app/src/main/res/layout/fragment_movies_details.xml

```diff
@@ -0,0 +1,183 @@
+            app:layout_constraintRight_toLeftOf="@id/guideline_margin_end"
+            app:layout_constraintTop_toBottomOf="@id/tv_name" />
+
+        <ru.androidacademy.gooseeye.RatingBarSvg
+            android:id="@+id/ratingBar"
+            style="@style/rating_bar_details_style"
+            android:layout_width="wrap_content"
+            android:layout_height="14dp"
+            android:numStars="5"
+            android:progressDrawable="@drawable/rating_bar_details"
+            android:rating="4"
+            android:stepSize="0.01"
+            android:layout_marginTop="7dp"
+            app:layout_constraintLeft_toRightOf="@id/guideline_margin_start"
+            app:layout_constraintTop_toBottomOf="@id/tv_tag_line"/>
+
+
+        <TextView
+            android:id="@+id/tv_review"
+            style="@style/review_style"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="6dp"
+            android:layout_marginTop="4dp"
+            android:fontFamily="sans-serif-black"
+            android:text="@string/review_text"
+            app:layout_constraintLeft_toRightOf="@id/ratingBar"
+            app:layout_constraintRight_toLeftOf="@id/guideline_margin_end"
+            app:layout_constraintTop_toBottomOf="@id/tv_tag_line" />
+
+        <TextView
+            android:id="@+id/tv_story_line"
+            style="@style/story_line_cast_style"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="24dp"
+            android:fontFamily="sans-serif-black"
+            android:text="@string/story_line_text"
+            app:layout_constraintLeft_toRightOf="@id/guideline_margin_start"
+            app:layout_constraintRight_toLeftOf="@id/guideline_margin_end"
+            app:layout_constraintTop_toBottomOf="@id/tv_review" />
+
+        <TextView
+            android:id="@+id/tv_story_line_details"
+            style="@style/white_14sp"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="4dp"
+            android:paddingTop="10dp"
+            android:paddingBottom="10dp"
+            android:text="@string/story_line_details_text"
+            app:layout_constraintLeft_toRightOf="@id/guideline_margin_start"
+            app:layout_constraintRight_toLeftOf="@id/guideline_margin_end"
+            app:layout_constraintTop_toBottomOf="@id/tv_story_line" />
+
+        <TextView
+            android:id="@+id/tv_cast"
+            style="@style/story_line_cast_style"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="24dp"
+            android:layout_marginBottom="7dp"
+            android:fontFamily="sans-serif-black"
+            android:paddingTop="3dp"
+            android:paddingBottom="3dp"
```

Comment:
Material design say, that you should use dimensions, which divider by 4.

Reply:
yea but it seems like there is such padding to match the design, right?